scripts/network-json-validator

```diff
@@ -431,16 +434,22 @@ def validate_networks databag
     end
   end
 
+  ip_versions = Hash.new(0)
   networks.each do |name, value|
     begin
       net = CrowbarNetwork.new(name, value)
+      ip_versions[net.ip_version] += 1
       net.validate
       @networks[name] = net
     rescue Exception => e
       raise "Cannot validate definition for network '#{name}': #{e.to_s}"
     end
   end
 
+  if ip_versions.length > 1
+    raise "All networks must either IPv4 or IPv6. A mix is not supported"
+  end
```

Comment:
Nice! You could even print out a statistic, sort of like: "You specified 3 IPv4 and 2 IPv6 networks. That's crazy!" :wink: